backend/core/src/migration/9999999999000-add-external-table-indexes.ts

```diff
@@ -0,0 +1,32 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+// REMOVE_WHEN_EXTERNAL_NOT_NEEDED
+export class externalListingIndexes9999999999000 implements MigrationInterface {
+  name = "externalListingIndexes9999999999000"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+
+    /*
+    // For filtering by num_bedrooms
+    await queryRunner.query(
```

Comment:
are these intentionally commented out?

Reply:
Whoops!  Good catch.  I had to comment them out temporarily when I regenerated the migrations with the split query and I forgot to add them back.  Fix inbound